browser/admin/src/integrator/AdminIntegratorSettings.ts

```diff
@@ -1520,9 +1521,20 @@ class SettingIframe {
 			);
 		} else {
 			// If user doesn't have any xcu file, we generate with default settings...
-			this.xcuEditor = new (window as any).Xcu('documentView.xcu', null);
-			this.xcuEditor.generateXcuAndUpload();
-			return await this.fetchAndPopulateSharedConfigs();
+			try {
+				if (!this.xcuInitializationAttempted) {
+					this.xcuInitializationAttempted = true;
+					this.xcuEditor = new (window as any).Xcu('documentView.xcu', null);
+					await this.xcuEditor.generateXcuAndUpload();
+					return await this.fetchAndPopulateSharedConfigs();
+				} else {
+					document.getElementById('xcu-section')?.remove();
+					console.warn('XCU file not found and automatic creation failed.');
+				}
+			} catch (error) {
+				console.error('Something went wrong while generating xcu file:', error);
```

Comment:
```suggestion
				console.error('Something went wrong while generating or uploading xcu file:', error);
```

It is more likely to be due to uploading files.
Generating files is implemented by ourselves in browser.